src/fundus/parser/data.py

```diff
@@ -16,10 +16,13 @@
 
 from typing_extensions import TypeAlias
 
-from fundus.logging import basic_logger
+from fundus.logging import create_logger
+
+__module_logger__ = create_logger(__name__)
```

Comment:
This is preference, but I would keep the variable name for the logger simple. Just `logger` should be enough. IMO, seeing the `__` everywhere clutters the code, as the logger is not related to any magic/dunder methods.